app/cli/src/main/scala/org/bitcoins/cli/ConsoleCli.scala

```diff
@@ -242,6 +243,14 @@ object ConsoleCli {
                 case accept: AcceptDLCOfferFromFile =>
                   accept.copy(path = path)
                 case other => other
+              })),
+          arg[Path]("destination")
+            .required()
```

Comment:
since these are both required, is there a reason the field is an `Option`?

Reply:
oops, made optional